test/test_dnf_content_plugin.py

```diff
@@ -6,13 +6,16 @@
 import os
 import types
 
-from nose.plugins.skip import SkipTest
+import pytest
 
+skip: bool = False
+import_error: str = ""
 try:
     import dnf
     import librepo
 except ImportError as e:
-    raise SkipTest(e)
+    skip = True
+    import_error = str(e)
```

Comment:
hm what about using `pytest.skip(msg, allow_module_level=True)` to skip the test from a file level? ie something like:
```python
try:
    import dnf
    import librepo
except ImportError as e:
    pytest.skip(f"DNF dependency could not be imported: {e}", allow_module_level=True)
```
this way, we do not need to decorate new classes added to this test module

Reply:
Ok.